Off-Top-App/off-top-python
# created an API to get each users session duration

**What I did:**

- created a restful API to find each users session duration by utilizing the API 'get-all-sessions'

**How to test:**

- You can use `http://localhost:5000/get-user-session-duration` on postman or visit it in your browser to see the list of JSON objects that appear which hold the user_id and their session_duration.
